src/Compilers/VisualBasic/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.vb

```diff
@@ -556,6 +556,163 @@ True
 ]]>
 
             Dim verifier As CompilationVerifier = CompileAndVerify(source, expectedOutput)
```

Comment:
Before this change, the inserted method prologue wasn't going through local rewriting except for async method. @AlekseyTs pointed out that codegen would take the tree and generate IL as is. That'd explain why it didn't fail for non-async method before (although I'm not familiar with codegen to confirm this myself).